Docs/source/running_cpp/parameters.rst

```diff
@@ -672,6 +672,9 @@ Numerics and algorithms
     See `this section of the FFTW documentation <http://www.fftw.org/fftw3_doc/Planner-Flags.html>`__
     for more information.
 
+* ``pstad.v_galilean`` (3 floats, in units of the speed of light)
+    Defines the galilean velocity.
```

Comment:
Thanks for updating the documentation. Could you add a bit more info:
- Please mention that the default is 0, and that a non-zero `v_galilean` activates the Galilean algorithm, which suppresses the Numerical Cherenkov Instability in boosted-frame simulation, and add:
```
as explained in :doc:`../theory/boosted_frame`
```
- Please mention that this requires the code to be compiled with `USE_PSATD=TRUE`
- We could also mention that this requires the `direct` deposition (does not work with the Esirkepov algorithm).

Reply:
Done in dd39e451e85.